### DISCUSSION TITLE
Logging out WhatsApp Client Due to Missmatch Authentication

### DISCUSSION THREAD

**Discussion by devalexandre** [dimaskiddo/go-whatsapp-multidevice-rest#36] [Q&A]
After sometime did auto logout :(

I'm using my phone number like username, and not found in wiki about this message

**Comment by ibnux** [comment]
is this latest version?

if updating from previous version, use `make clean` then `make vendor` before running it or build it

sometimes network too

**Comment by dimaskiddo** [comment]
Probably the ID not Registered with same number or like pak @ibnux said network issue when trying to checking the existance of the ID 🤔
